### PR TITLE
Feature/deb compatlevel13

### PR DESCRIPTION
**Pull request by arminfelder**

### CODE REVIEW

**Comment by versat** [comment]
@fralken Can you also have a look and maybe test this PR in your environment?

File: ntlm.c [hunk 1]
```diff
@@ -327,6 +380,7 @@ int ntlm_response(char **dst, char *challenge, int challen, struct auth_s *creds
 #endif
 	char *buf;
 	char *udomain;
+    char *target;
```
Comment (versat) [inline ntlm.c:383]: indentation in this file is different, spaces vs tabs

**Comment by arminfelder** [comment]
@fralken yes sorry, I think I messed it up by working at night..., this pull request should only contain changes regarding the deb packages

**Comment by versat** [comment]
Ok, yes we can test and merge PR #19 first then.
The changes here then should be minimal.

**Comment by fralken** [comment]
An idea could be to create a new PR (or modify this one) cherry picking only commit 5a47622.

**Comment by versat** [comment]
> An idea could be to create a new PR (or modify this one) cherry picking only commit [5a47622](https://github.com/versat/cntlm/commit/5a4762238cae72172563ea9341d4e929b86d470f).

Yes I was thinking nearly the same.

**Comment by arminfelder** [comment]
@versat thanks for cleaning up my mess, haven't had the time to do it

**Comment by versat** [comment]
> @versat thanks for cleaning up my mess, haven't had the time to do it

No problem, thanks for the PR 🙂

**Pull request merged**